code/datums/components/pellet_cloud.dm

```diff
@@ -27,6 +27,7 @@
 	var/list/bodies /// For grenades, any /mob/living's the grenade is moved onto, see [/datum/component/pellet_cloud/proc/handle_martyrs()]
 	var/list/purple_hearts /// For grenades, tracking people who die covering a grenade for achievement purposes, see [/datum/component/pellet_cloud/proc/handle_martyrs()]
 
+	var/pellet_delta /// For grenades, tracking how many pellets are removed due to martyrs and how many pellets are added due to the last person to touch it being on top of it
```

Comment:
this is not how you doc, whoever did this is cringe.

http://codedocs.tgstation13.org/datum/component/pellet_cloud.html

it MUST be
```DM
///thing doc here
var/thing